src/ui/public/vislib/lib/layout/layout_types.js

```diff
@@ -13,10 +13,8 @@ export default function LayoutTypeFactory(Private) {
    * @return {Function} Returns an Object of HTML layouts for each visualization class
    */
   return {
-    histogram: Private(VislibLibLayoutTypesColumnLayoutProvider),
-    line: Private(VislibLibLayoutTypesColumnLayoutProvider),
-    area: Private(VislibLibLayoutTypesColumnLayoutProvider),
     pie: Private(VislibLibLayoutTypesPieLayoutProvider),
-    tile_map: Private(VislibLibLayoutTypesMapLayoutProvider)
+    tile_map: Private(VislibLibLayoutTypesMapLayoutProvider),
+    point_series: Private(VislibLibLayoutTypesColumnLayoutProvider)
```

Comment:
Is it safe to assume that point_series is now a base layout type for histogram, line, and area layouts?